projects/Network Usage Tracker/requirements.txt

```diff
@@ -0,0 +1,6 @@
+libraries used : tkinter
+		 messagebox
```

Comment:
please use pip freeze list>requirements.txt to generate requirement file in standard format